demos/chat-client/chat-web-ui/script.js

```diff
@@ -0,0 +1,127 @@
+function init() {
+
+    //--------------------------- Important Variables----------------------------
+    var inactiveMessage = "Server is down, Please contact the developer to activate it"
+    chatPopup = document.querySelector(".chat-popup")
+    chatBtn = document.querySelector(".chat-btn")
+    chatSubmit = document.querySelector(".chat-submit")
+    chatHeader = document.querySelector(".chat-header")
+    chatArea = document.querySelector(".chat-area")
+    chatInput = document.querySelector(".chat-input")
+    expandWindow = document.querySelector(".expand-chat-window")
+    root = document.documentElement;
+    chatPopup.style.display = "none"
+    var host = "http://localhost:8080"
```

Comment:
This should:
1. Use something more unique. I suggested port `50010` in #69 
2. Use `https`